File: spec/python/test_str_eos.py
import unittest

from str_eos import StrEos

class TestStrEos(unittest.TestCase):
    def test_str_eos(self):
        with StrEos.from_file("src/term_strz.bin") as r:

            self.assertEqual(r.str, "foo|bar|baz@")

File: spec/lua/test_str_eos.lua
local luaunit = require("luaunit")

require("str_eos")

TestStrEos = {}

function TestStrEos:test_str_eos()
    local r = StrEos:from_file("src/term_strz.bin")

    luaunit.assertEquals(r.str, "foo|bar|baz@")
end
